qiskit/synthesis/linear_phase/cx_cz_depth_lnn.py

```diff
@@ -33,7 +33,7 @@
 
 from qiskit.circuit import QuantumCircuit
 from qiskit.synthesis.linear.linear_matrix_utils import calc_inverse_matrix
-from qiskit.synthesis.linear.linear_depth_lnn import _optimize_cx_circ_depth_5n_line
+from qiskit._accelerate.synthesis.linear import optimize_cx_circ_depth_5n_line
```

Comment:
is this change needed, i.e. shouldn't the relevant function be importable from ``qiskit.synthesis.linear.linear_depth_lnn``?

Reply:
This is not the "main" function in the sense that it is not the one that generates the circuit, but the helper method that generates the instruction lists (as lists of pairs), so I'm not sure if we should wrap it as well.